lib/simple_lru.js

```diff
@@ -43,10 +44,12 @@ Cache.prototype.get = function(key,hit){
      * it to the head of linked list  
      */
     hit = hit != undefined && hit != null ? hit : true;
-    if(cacheVal && hit)
+
+    if(cacheVal && hit && (!cacheVal.time || (Date.now() - cacheVal.time) < this.maxTime)) 
```

Comment:
I did that at first, but probably you will set it with an updated item right after the get function return undefined.

And there still gonna be expired item in the cache that we haven't checked. 

So I think it's a little bit better to leave it there and it might be pushed out if it isn't used.